src/data/complex/is_R_or_C.lean

```diff
@@ -264,10 +275,16 @@ lemma im_sq_le_norm_sq (z : K) : im z * im z ≤ norm_sq z :=
 le_add_of_nonneg_left (mul_self_nonneg _)
 
 theorem mul_conj (z : K) : z * conj z = ((norm_sq z) : K) :=
-by simp [ext_iff, norm_sq, mul_comm, sub_eq_neg_add, add_comm]
+by simp only [map_add, add_zero, is_R_or_C.ext_iff, is_R_or_C.mul_re, monoid_with_zero_hom.coe_mk,
+              add_left_inj, mul_eq_mul_left_iff, zero_mul, add_comm, true_or, eq_self_iff_true,
+              is_R_or_C.mul_im, is_R_or_C.of_real_re, mul_neg_eq_neg_mul_symm, add_right_neg,
+              is_R_or_C.conj_re, zero_add, is_R_or_C.norm_sq.equations._eqn_1, mul_comm, and_self,
```

Comment:
Hmm, that explicit equation lemma looks quite ugly. Does `is_R_or_C.norm_sq` not work?